department-of-veterans-affairs/caseflow
# Update caseflow-frontend-toolkit reference to include the lastest commit

Resolves  https://github.com/department-of-veterans-affairs/caseflow/issues/8893

### Connects to
https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/pull/45
https://github.com/department-of-veterans-affairs/caseflow-feedback/pull/197


### Testing Plan
1. Switch user to BVAAABSHIRE
2. Go to http://localhost:3000/queue/appeals/2362049
3. Click on Send Feedback link in footer
4. Also, on http://localhost:3000/queue/appeals/2362049 page, click on Send Feeback link in Dropdown menu
5. Make sure that in both 3. and 4. the link has original_url param with current page url 

![feedbackfooter](https://user-images.githubusercontent.com/4960030/52295740-8bb93400-294a-11e9-823a-71723895fa2a.gif)

![feedbacknavbar](https://user-images.githubusercontent.com/4960030/52295815-b99e7880-294a-11e9-92a6-ee34be1e208d.gif)
